examples/docker/MLproject

```diff
@@ -3,6 +3,11 @@ name: docker-example
 docker_env:
   image:  mlflow-docker-example
 
+kubernetes_env:
+  job_namespace: mlflow
+  image_namespace: marcusrehm
```

Comment:
Hm, is there something special about the URI structure `registry/namespace/name`? Would it be more simple to ask the user to provide just an `image_name_prefix` or something?

Reply:
hey @andrewmchen , thanks for your comments!
Actually, as I said above to @talgo10, now it's possible to pass a template file for the job. There is a simple one inside the docker example folder.

Reply:
About the `job_namespace`, it's used to specify in which k8s namespace the job show be deployed. I think the job namespace where the training will be executed would be more of a project definition than a users choice, that's why I let it in the MLproject file.

Reply:
Complementing, as I said above, regarding the `job_namespace`, I think the same applies to publish images. So the registry and image namespace would be in MLproject file.

Reply:
`job_namespace` is not generalizable to all users right? If you say this project should be run in `namespace = marcusrehm` and I don't have this namespace in my k8s cluster, the project run will fail.

Same with `registry` and `image_namespace`: if you choose a registry I don't have push access to my run will fail.

Reply:
hey @andrewmchen ! I got your point. I was thinking about our process here (which led me to develop the feature) and your approach is better and covers more use cases.

Reply:
Awesome, BTW I just wanted to make sure you knew about this refactoring that @aarondav  is doing to get ready for 1.0. https://github.com/mlflow/mlflow/pull/1222/files#diff-67cc4fcc56ad80e1028ac5d959b05985R71

It might make sense to put the `k8s_context`, `k8s_job_template`, and `docker_registry` fields in that file as well.

Reply:
I did the update in my branch. I'm still doing the changes but right now this commit is already merged. :)